plugins/inputs/linux_fscache/linux_fscache.go

```diff
@@ -0,0 +1,67 @@
+package linux_fscache
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type FSCache struct {
+	statFile string
+}
+
+func (f *FSCache) Description() string {
+	return "Get FS-Cache statistics from /proc/fs/fscache/stats"
+}
+
+func (f *FSCache) SampleConfig() string { return "" }
+
+func (f *FSCache) Gather(acc telegraf.Accumulator) error {
+
+	s, err := os.Stat(f.statFile)
+	if err != nil {
+		return err
+	} else if s.Size() == 0 {
+		return fmt.Errorf("fscache: %s zero length!", f.statFile)
+	}
+
+	data, err := ioutil.ReadFile(f.statFile)
+	if err != nil {
+		return err
+	}
+
+	lines := strings.Split(string(data), "\n")
+	fields := make(map[string]interface{})
+	for i := 1; i < len(lines); i++ {
+		parts := strings.Split(lines[i], ":")
+		prefix := strings.TrimSpace(parts[0])
+		subparts := strings.Split(parts[1], " ")
+		for j := 0; j < len(subparts); j++ {
+			values := strings.Split(strings.TrimSpace(subparts[j]), "=")
+			if len(values) == 2 {
+				v, err := strconv.ParseInt(values[1], 10, 64)
+				if err != nil {
+					return err
+				}
+				fields[prefix+"_"+values[0]] = int64(v)
```

Comment:
To match with Telegraf convention, could you call `internal.SnakeCase(values[0])` here so that the fields come out like, for example, `attr_chg_n`.

Reply:
done.